src/sentry/services/hybrid_cloud/auth/__init__.py

```diff
@@ -84,24 +83,33 @@ def authenticate(self, request: Request) -> Tuple[Any, Any] | None:
         return None
 
 
-class AuthService(InterfaceWithLifecycle):
-    @abc.abstractmethod
+class AuthService(RpcService):
+    name = "auth"
+    local_mode = SiloMode.CONTROL
+
+    @classmethod
+    def get_local_implementation(cls) -> RpcService:
+        from sentry.services.hybrid_cloud.auth.impl import DatabaseBackedAuthService
+
+        return DatabaseBackedAuthService()
+
+    @rpc_method
     def authenticate(self, *, request: AuthenticationRequest) -> MiddlewareAuthenticationResponse:
         pass
 
-    @abc.abstractmethod
+    @rpc_method
```

Comment:
One minor nit and thought: is it possible to leave the `abc.abstractmethod` in place?  Rather than `rpc_method` imply abstractmethod, it'd be nice if it *requires* abstract method (even check that it is) and require you to use both decorators explicitly.  But this is definitely in the category of minor nits and debatable, not critical.

Reply:
Would you believe I initially wrote it that way (with the check, even) and then changed it? 😅 But yeah, I don't have a strong preference and I think your way is good. `abstractmethod` is enough of a quasi-language-level element that it could help readability a lot to leave it in.